src/controllers/scenario/index.ts

```diff
@@ -41,12 +42,9 @@ export class scenario extends Controller implements IScenario {
   }
   
   @Get('{executable}')
-  public async get(@Path() executable: Executables): Promise<ExamplesResult> {
-    this.log.debug(`attempting to execute ${executable} scenario`)
+  public async get(@Path() executable: Executables , @Query() params: string[]): Promise<HostResponse> {
+    this.log.debug(`attempting to execute ${executable} scenario with [${params}] arguments`)
     
-    return ({
-      cheri: await this.execute(`${executable}-cheri`),
-      aarch64: await this.execute(`${executable}-aarch64`),
-    })
+    return this.execute(`${executable}-cheri ${params}`)
```

Comment:
how does an `aarch64` binary execute?

Reply:
😮‍💨 
M<... this is wrong! let me update. thanks for flagging up

Reply:
This has been resolved. It should have been only `${executable}` arch comes from ui